### ISSUE TITLE
NTR: Tropheryma whipplei bacteria in CSF (Lab: cerebrospinal fluid tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8479]
New term request Tropheryma whipplei bacteria in CSF (segal_170417124948): Cerebrospinal fluid

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000766
- label: Positive CSF Tropheryma whipplei bacteria microscopy
- parents: [Positive CSF infectious agent test (HP:0430123)](https://hpo.jax.org/app/browse/term/HP:0430123)
- pmid: PMID:28298472
- def: Demonstration of Tropheryma whipplei bacteria in the cerebrospinal fluid (CSF) my microscopic analysis.
- comment: T. whipplei is a rod-shaped, Gram-positive bacterium.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8479
- synonyms:
